cdx.go

```diff
@@ -0,0 +1,270 @@
+package warc
+
+import (
+	"bufio"
+	"fmt"
+	"mime"
+	"net"
+	"net/http"
+	"net/url"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const (
```

Comment:
golint will be angry at the following:
`cdx.go:18:2: exported const CDXCanonizedURL should have comment (or a comment on this block) or be unexported`